### ISSUE TITLE
jp2-create in -test is following the wrong path

### ISSUE THREAD

**Issue by blalbrit** [sul-dlss/assembly#41]
jp2-create : input file /dor/assembly/ck/796/mp/0576/ck796mp0576/ck796mp0576_img_1.jpg does not exist

Should be /dor/assembly/ck/796/mp/0576/ck796mp0576/content/ck796mp0576_img_1.jpg

**Comment by peetucket** [comment]
the robot should be looking in either location and only fail if it can't be found anywhere...is it actually failing to find a legitimate file?

**Comment by blalbrit** [comment]
Yes - please see path in initial description. However, to push the content through I did copy it all up one level - leaving the originals in place.

**Comment by peetucket** [comment]
Actually, that makes sense to me... the two expected paths for content are either:

OLD STYLE (content and metadata files mixed into one folder at end of druid tree): 
/dor/assembly/ck/796/mp/0576/ck796mp0576_img_1.jpg  
/dor/assembly/ck/796/mp/0576/contentMetadata.xml

NEW STYLE (content and metadata in separate folders under a folder with full druid): 
/dor/assembly/ck/796/mp/0576/ck796mp0576/content/ck796mp0576_img_1.jpg
/dor/assembly/ck/796/mp/0576/ck796mp0576/metadata/contentMetadata.xml

It looks like you had content at at same level as the druid tree folder itself and not in the "content" subfolder?  Or am I misunderstanding?

By default, pre-assembly will stage into the new style (unless you set new_druid_tree_format:  false in the YAML config file for that run)
